dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/intermediate/group/ComponentGroup.kt

```diff
@@ -19,6 +21,7 @@ class ComponentGroup(
 ) : ComponentBase(identifier, parent), FieldNodeParent, ComponentGroupApi by componentGroupApi {
 
     var viewPageLabelBadgeColor: LabelBadgeColor? = null
+    var uploadPageLabelBadgeColor: LabelBadgeColorUpload? = null
```

Comment:
Do we need a seperate enum type for `LabelBadgeColorUpload`. Are they not using the same color-palette anyway?

Reply:
Should be the same. Let's remove

Reply:
Done